README.md

```diff
@@ -58,6 +58,20 @@ $ curl http://0.0.0.0:8080/record/23
 ## Next.js app
 When you run for the first time, you will need to copy the template .env.example file and add missed variables:
 
-   ```sh
-   $ cp .env.example .env
-   ```
\ No newline at end of file
+```sh
+$ cp .env.example .env
```

Comment:
I think we should also update the content of this file, because right now there is incorrect API value set to: 
```NEXT_PUBLIC_API_URL=http://data-api:8080```

it should be 

```NEXT_PUBLIC_API_URL=http://localhost:8080```

Reply:
fixed